fv3config/_datastore.py

```diff
@@ -57,9 +65,9 @@ def get_orographic_forcing_directory(config):
     """Return the string path of the orographic forcing directory specified by a config dictionary.
     """
     resolution = get_resolution(config)
-    dirname = os.path.join(local_archive_dir, f'orographic_data/{resolution}')
+    dirname = os.path.join(LOCAL_ARCHIVE_DIR, f'orographic_data/{resolution}')
```

Comment:
Should this be `get_cache_dir()`? Or why do you sometimes use `LOCAL_ARCHIVE_DIR` and sometimes `get_cache_dir()`?

Reply:
I've changed it to uniformly use `get_cache_dir()`.